### ISSUE TITLE
Created two bugs with similar names, No similar items appear

### ISSUE THREAD

**Issue by nbdb** [ostreifel/duplicate-bug#6]
I have a testing project to test new extensions before we include them in our development or bug reporting progress.

Both user stories and bugs with similar, or even identical titles, in the same area and iteration don't show any similar work items.

**Comment by ostreifel** [comment]
Yeah I've noticed that too, the more I think about it the more I think that an extension is the wrong way to implement a duplicate bug checker given the limited resources available in the users browser - limited cpu, small bandwidth to check existing work items, not having the search api exposed to extensions etc. 

For that reason, as well as losing interest in the concept, I am unpublishing the extension from the marketplace unless I think of some radically better way to implement the idea at some point in the future.

**Comment by nbdb** [comment]
Unfortunate, but that makes sense. Thanks for the response though!
